cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +411,88 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("n3000", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
+				Skip("Cluster doesn't have intel fpga n3000 nic")
+			}
+		})
+
+		It("should have at least 1 intel n3000 nic", func() {
+			n3000NicExists := false
+			for _, n3000Node := range n3000NodeList.Items {
+				if n3000NicExists != true {
+					for _, fortville := range n3000Node.Status.Fortville {
+						if n3000NicExists != true {
+							for _, nic := range fortville.NICs {
+								if strings.Contains(nic.Name, "N3000") {
+									n3000NicExists = true
+									break
```

Comment:
If you use a label break you can get rid of all the nicexists != true and break the outer loop:
https://www.ardanlabs.com/blog/2013/11/label-breaks-in-go.html

Reply:
Cool, thanks for tip!